src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/gtfs/RouteTestsIT.java

```diff
@@ -0,0 +1,131 @@
+import fi.otavanopisto.kuntaapi.server.integrations.ptv.PtvConsts;
+import fi.otavanopisto.kuntaapi.test.AbstractIntegrationTest;
+
+public class RouteTestsIT extends AbstractIntegrationTest {
+  
+  /**
+   * Starts WireMock
+   */
+  @Rule
+  public WireMockRule wireMockRule = new WireMockRule(getWireMockPort());
+  
+  @Before
+  public void beforeTest() throws InterruptedException {
+    createPtvSettings();
+    
+    getPtvMocker()
+      .mockOrganizations("0de268cf-1ea1-4719-8a6e-1150933b6b9e")
+      .startMock();
+
+    waitApiListCount("/organizations", 1);
+    
+    createGtfsSettings(getOrganizationId(0));
+
+    waitApiListCount(String.format("/organizations/%s/transportRoutes", getOrganizationId(0)), 3); 
+  }
+
+  @After
+  public void afterClass() {
+    String organizationId = getOrganizationId(0);
+    getPtvMocker().endMock();
+    deletePtvSettings();
+    deleteGtfsSettings(organizationId);
+  }
+
+  @Test
+  public void testListRoutes() {
+    String organizationId = getOrganizationId(0);
+    
+    Response response = given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportRoutes", organizationId);
+    
+    response
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(3))
+      .body("id[1]", notNullValue())
+      .body("shortName[1]", is("1 A"))
+      .body("longName[1]", is("Tori-Tuppurala-Tori"))
+      .body("agencyId[1]", notNullValue());
+    
+    String agencyId = response.body().jsonPath().getString("agencyId[1]");
+    assertFound(String.format("/organizations/%s/transportAgencies/%s", organizationId, agencyId));
+  }
+  
+  @Test
+  public void testFindRoute() {
+    String organizationId = getOrganizationId(0);
+    Response response = given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportRoutes/{routeId}", organizationId, getOrganizationRouteId(organizationId, 2));
+    
+    response
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("shortName", is("1 AB"))
+      .body("longName", is("Tori-Tuppurala-Tori"))
+      .body("agencyId", notNullValue());
+    
+    String agencyId = response.body().jsonPath().getString("agencyId");
+    assertFound(String.format("/organizations/%s/transportAgencies/%s", organizationId, agencyId));
+  }
+  
+  @Test
+  public void testOrganizationRoutesNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationRouteId = getOrganizationRouteId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/transportRoutes/%s", organizationId, organizationRouteId));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "/organizations/%s/transportRoutes/%s" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)